frontend/require/NavView.js

```diff
@@ -3,7 +3,7 @@ define(['underscore', 'backbone', 'mustache', 'text!NavView.html'], function(_,
 
     var NavView = Backbone.View.extend({
 
-        tagName: 'nav',
+        tagName: 'div',
```

Comment:
Ah, I get it now! I was thinking of the `tagName: 'nav'` as being the same thing as the top-level `<nav>` in `NavView.html`, but of course the tag will actually be a new enclosing element. Thanks for explaining!